tox.ini

```diff
@@ -18,7 +18,7 @@ setenv =
 [testenv:pylint]
 deps =
   pylint_quotes
-  pylint
+  pylint <= 2.7.4
```

Comment:
an odd issue with pyyaml quotes is causing me to not use the most recent version of pylint